source/backend/api/Areas/Admin/Controllers/ClaimController.cs

```diff
@@ -95,66 +93,6 @@ public IActionResult GetClaim(Guid key)
             var claim = _mapper.Map<Model.ClaimModel>(entity);
             return new JsonResult(claim);
         }
-
```

Comment:
dead controller methods, we do not allow roles/claims to be edited in MOTI PIMS.